Hide special workspaces in hyprland workspaces, and changes to hyprland taskbar

saipavanc opened the pull request:
- Fix bug #380 in hyprland workspaces
- Make following changes to hyprland taskbar
-- Add options `all-workspaces` and `show-workspace` to make it possible to hide workspaces and limit clients to current workspace.
-- ~The context menu for clients now shows existing workspaces, than just a list of 1-10. Any other inactive workspaces are also listed if the setting `show-inactive-workspaces` are enabled. Which is enabled by default.~

saipavanc commented:
Along with the bug fix, I also made some similar changes to Hyprland taskbar, which allows it to take workspaces from workspace rules for the context menu. So similar options follow here as well.

nwg-piotr commented:
Don't touch the HyprlandTaskbar module, please. I want the context menu to stay as is.

saipavanc commented:
It doesn't affect the default functioning. I request you to please take a look! Just added a couple of settings which gives it more flexibility.

saipavanc commented:
In the above commit, I tested all the options again, and removed an unnecessary option I added to taskbar. Please check, this should now have the intended default functionality.

nwg-piotr commented:
> Please check, this should now have the intended default functionality.

Sorry, it has not. I need to mark "All outputs" to see all workspaces in the context menu. I don't want to see tasks from all outputs in this certain panel. I just want to be able to right-click and send a task to any workspace of my choice.
